lib/krikri/search_results_helper_behavior.rb

```diff
@@ -42,5 +42,51 @@ def render_bookmarks_control?
       false
     end
 
+    # Render enriched record for view
+    # @param [Krikri::SearchIndexDocument]
+    # @return [String]
+    def render_enriched_record(document)
+      agg = document.aggregation
+      return error_msg unless agg.present?
+      JSON.pretty_generate(agg.to_jsonld)
```

Comment:
Thinking out loud - do we want to present more API-like data here (e.g, with a briefer/differently mapped context)?